### DISCUSSION TITLE
Rendering "artifacts" when zooming and then panning in TileLayer

### DISCUSSION THREAD

**Discussion by ffendt** [visgl/deck.gl#8820] [Q&A]
When working with TiledLayer, I observed that deck.gl will somehow remember what it previously rendered for a portion of the map and render that until new information is available.

![Bildschirmaufnahme 2024-04-23 um 14 20 09](https://github.com/visgl/deck.gl/assets/2587759/ea2f05f5-9032-4e3e-8dc1-bd179e9813cd)

Since I don't know how to explain this properly, I created a small gif that should show the behavior. I'm zooming into the map and then move around the map. At some point you can see, that deck.gl shows tiles of the previous, lower zoom level until the new data has been loaded.

Why is this happening? Is there some configuration for this?

I saw that there is the `refinementStrategy` (`-> 'never'`) that seems to work quite good, but I also have this behavior when working with multiple TileLayers that are hidden using the `visible` property. E.g. a layer that is only shown until zoom 10 (`visible: zoom <= 10`) and a layer that is shown above (`visible: zoom > 10`). When zooming in from say 9 to 12 and then moving around the map, I can see artifacts from the `<=10`-layer.

Is there some sort of view cache that includes layers that have previously been visible for a given viewport? Is there someway to disable that cache?

Note: I produced the behavior on the [TileLayer API reference page](https://deck.gl/docs/api-reference/geo-layers/tile-layer) by throttling my connection to 3G using the browser dev tools.

**Comment by ibgreen** [comment]
Do I understand correctly that you are unhappy with to the pixelated look when showing the parent tile? Would you rather have a blank area on the map when panning / zooming in rather than show cached parent tiles? 

I.e. the question is what to show while all the current zoom level tiles load?

**Reply by ffendt** [reply]
I had the feeling, that this also happens with layers that should no longer be displayed at all (i.e. having `visible: false`). But actually, when creating a reproducer in Code Sandbox, I found out that there is a bug in my original code and everything works as I'd expect it to work.
So: Thanks for your fast answer and sorry for the misunderstanding. I'm really happy with deck.gl and the efforts you and the other contributors put into it.
